.gitlab-ci.yml

```diff
@@ -229,6 +229,26 @@ build:bleeding-edge:
     EXTRA_CONF: "-native-compiler yes -coqide opt"
     EXTRA_PACKAGES: "$COQIDE_PACKAGES"
 
+build:windows:
+  stage: test
+  artifacts:
+    name: "$CI_JOB_NAME"
+    paths:
+      - dev\nsis\*.exe
+      - coq-opensource-archive-windows-*.zip
+    expire_in: 1 week
+  dependencies: []
+  tags:
+    - windows
+  before_script:
+    - SET ARCH=64
+  script:
+    - powershell -Command "(New-Object Net.WebClient).DownloadFile('http://www.cygwin.com/setup-x86_64.exe', 'setup-x86_64.exe')"
```

Comment:
IIRC the download and installation of dependencies take a long time in the whole packaging process. In the future, we could end up saving a lot of time by using a personalized Docker image, especially if it is cached on the runners. (Not a request for this PR.)

Reply:
How does Docker work on Windows?

Reply:
No experience with it but it seems to be supported.